bin/cmds/app/add-types.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const Log = require('../../..').Log;
+const App = require('../../..').App;
```

Comment:
Is dit niet consistenter met wat we eigenlijk altijd doen? 
```suggestion
const { App, Log } = require('../../..');
```

Reply:
yea behalve dat node-homey wat oudere code is en dat heel consistent niet zo doet...
Ben het er mee eens dat destructuring mooier is though